### PR TITLE
Corrected Misleading Texts in Sentence

### PR DESCRIPTION
**Pull request by AjaiDubey**
The sentence was started with a Past tense and making it hard in between to understand what it is saying. But now it is good.

NOTE: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributing

### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -187,8 +187,8 @@ If you are asked to update your patch by a reviewer, please do so! Remember:
 you are probably the one in need of it. You must be prepared to apply changes
 to it if necessary.
 
-If your Pull Request was accepted and fixes a bug, adds functionality, or
-makes it significantly easier to use or understand Kong, congratulations!
+If your Pull Request was accepted and it fixed a bug, added functionality, or
```
Comment (Tieske) [inline CONTRIBUTING.md]: I think it's easier if you remove the "it" here, eg "it fixed" => "fixed"

**Comment by AjaiDubey** [comment]
Tell me what are these failing checks?

**Comment by thibaultcha** [comment]
What is wrong with the current wording?

**Comment by thibaultcha** [comment]
Hi,

Thanks for the suggestions, but the current wording is fine as it is.

**Pull request closed without merging**